src/sparseml/yolov8/validators.py

```diff
@@ -30,9 +31,9 @@
 
 
 class SparseValidator(BaseValidator):
-    def __call__(self, trainer=None, model=None):
+    def __call__(self, trainer=None, model=None, training=True):
         # the **only** difference in this call is that we pass fuse=False to AutoBackend
```

Comment:
Can you remove this comment? No longer accurate and I think we have other changes planned for this function anyway